certification/pyxis/submit.go

```diff
@@ -68,6 +71,24 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *Certificatio
 		if err != nil {
 			return nil, fmt.Errorf("could not get image: %v", err)
 		}
+
+		// checking to see if the original value is certified and the previous value is not certified,
+		// this would indicate that a partner is running preflight again, and during the first run there was a timeout/error
+		// in a check that interacts with pyxis and we need to correct the certified value for the image
+		if certified && !certImage.Certified {
+			// instantiating an updateCertImage struct, when calling updateImage we need to provide the ID, architecture,
+			// and change the certified value to `true`
+			updatedCertImage := &CertImage{
```

Comment:
I would still do this in the method, not here.

Reply:
The only thing that I kept here is changing the certified flag to come from the original certImage, ie the one we create during the current execution. Then I pass in the struct that we got from the `GET` call to pyxis.